include/control_filters/low_pass_filter.hpp

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) 2021, Stogl Robotics Consulting UG (haftungsbeschränkt)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CONTROL_FILTERS__LOW_PASS_FILTER_HPP
+#define CONTROL_FILTERS__LOW_PASS_FILTER_HPP
+
+#include <cmath>
+#include "geometry_msgs/msg/wrench_stamped.hpp"
+#include "filters/filter_base.hpp"
+#include "filters/filter_chain.hpp"
+#include <Eigen/Dense>
+#include "rclcpp/node.hpp"
+
+
+namespace control_filters
+{
+template <typename T>
+class LowPassFilter : public filters::FilterBase<T>
+{
+public:
+    LowPassFilter();
+
+    ~LowPassFilter();
+
+    /** @brief Configure filter parameters  */
+    virtual bool configure();
+
+    bool configure(const std::string& ns);
+
+    virtual bool update(const T& data_in, T& data_out);
+
+    /** \brief Get most recent parameters */
+    bool updateParameters();
+
+private:
+    /** \brief Dynamic parameter callback activated when parameters change */
+    void parameterCallback();
+    rclcpp::Node::SharedPtr node_;
+    rclcpp::Logger logger_;
```

Comment:
Do we need this? Isn't there access to logging interface available through `FilterBase`?